docs/content/docs/recipes/teams.md

```diff
@@ -52,7 +52,7 @@ This tutorial assumes that you are familiar with the [Fluid Framework Overview](
 
 ### Defining Fluid functions and parameters
 
-1. Since we are building in the context of Microsoft Teams, having all Fluid related importations, initialization, and functions in one place will give better clarity and easier usage down the road. Add the following code below the import statements. The comments will defined all the functions and constants we'll need to interact with the Fluid service and container.
+1. Since this is building in the context of Microsoft Teams, having all Fluid related importations, initialization, and functions in one place will be easier to use down the road. Add the following code below the import statements. The comments will defined all the functions and constants needed to interact with the Fluid service and container.
```

Comment:
```suggestion
1. Since this app is intended to be used in the context of Microsoft Teams, having all Fluid related imports, initialization, and functions in one place makes it easier to use in the future. Add the following code below the import statements. The comments will define all the functions and constants needed to interact with the Fluid service and container.
```